shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -141,14 +141,37 @@ package object symbols {
    *
    * Refined by typer.
    */
-  case class CallTarget(name: Name, symbols: List[Set[BlockSymbol]]) extends BlockSymbol
+  case class CallTarget(name: Name, symbols: List[Set[BlockSymbol]]) extends Synthetic with BlockSymbol
+
+  /**
+   * A symbol that represents a termlevel capability
+   */
+  trait Capability extends BlockSymbol {
+    def effect: UserEffect
+    val name = effect.name
+  }
+  case class UserCapability(effect: UserEffect) extends Capability
+
+  case class StateCapability(effect: UserEffect, get: EffectOp, put: EffectOp) extends Capability
+  object StateCapability {
+    def apply(binder: VarBinder)(implicit C: Context): StateCapability = {
+      val tpe = C.valueTypeOf(binder)
+      val eff = UserEffect(binder.name, Nil)
+      val get = EffectOp(binder.name.rename(name => "get"), Nil, List(Nil), Some(tpe / Pure), eff)
+      val put = EffectOp(binder.name.rename(name => "put"), Nil, List(List(ValueParam(binder.name, Some(tpe)))), Some(builtins.TUnit / Pure), eff)
+      eff.ops = List(get, put)
+      StateCapability(eff, get, put)
+    }
+  }
 
   /**
    * Types
    */
   sealed trait Type
 
-  // like Params but without name binders
+  /**
+   * like Params but without name binders
```

Comment:
Is there a reason for this change? Should we use only multi-line comments or is this something arbitrary?

Reply:
I just changed to multiline, because it would actually be available as doc comment this way. We don't generate a documentation, but this could help in the future.